common/locales/en/spells.json

```diff
@@ -1,6 +1,6 @@
 {
   "spellWizardFireballText": "Burst of Flames",
-  "spellWizardFireballNotes": "Flames burst from your hands. You gain XP, and you deal extra damage to Bosses! (Based on: INT)",
+  "spellWizardFireballNotes": "Flames burst from your hands. You gain XP, and you deal extra damage to Bosses! Click on a task to cast. (Based on: INT)",
```

Comment:
@lemoness Do you think these instructions for clicking on the task should come before or after the parenthetical? I think they should come after, since the parenthetical is describing the skill, not the act of clicking on a task.